test/cram/test.t

```diff
@@ -0,0 +1,20 @@
+This is a cram test for the new executable.
+
+# Run the executable
+  $ dummy_terminal example.diff n n
+  Operation 2 of 14, 1 hunk
```

Comment:
There seem to be a problem here: Since two `n` are given, we would expect the operation number to be `3` (zero `n` is "Operation 1", one `n` is "Operation 2", two `n` is "Operation 3").

Reply:
This has been fixed.